src/external_cli/rustup.rs

```diff
@@ -0,0 +1,47 @@
+//! Utilities for the `rustup` CLI tool.
+
+#![expect(dead_code, reason = "Will be used for build/run commands")]
+
+use std::process::Command;
+
+use dialoguer::Confirm;
+
+const PROGRAM: &str = "rustup";
```

Comment:
Could we put this in an env var?

Reply:
Let's spin this up into a follow-up issue. (Related: #71.)

Reply:
Sure!